packages/@webex/internal-plugin-metrics/src/call-diagnostic/call-diagnostic-metrics.ts

```diff
@@ -511,6 +514,18 @@ export default class CallDiagnosticMetrics extends StatelessWebexPlugin {
       }
     }
 
+    if (isSdpOfferCreationError(rawError)) {
+      // error code is 30005, but that's not specific enough. we also need to check error.cause.type
+      const causeType = rawError.cause?.type || 'GENERAL';
+
+      return this.getErrorPayloadForClientErrorCode({
+        serviceErrorCode: undefined,
+        clientErrorCode: SDP_OFFER_CREATION_ERROR_MAP[causeType],
```

Comment:
SDP_OFFER_CREATION_ERROR_MAP has only entries for GENERAL and SDP_MUNGE_MISSING_CODECS, so when we get any other `cause.type` from WCME we will end up with `undefined` here and we probably would want 2050 in those cases, right?

Reply:
i wasn't sure about that... do we want the other ones to map to `2050` or each have their own specific code for each specific cause?

Reply:
if we get any error that's a `Errors.SdpOfferCreationError` no matter what `cause.type` it has, it would be better to map it to 2050, because that has an ErrorDescription of "SdpOfferCreationError" which is better than "unknown"

Reply:
makes sense to me. will change.